apps/api/.env-template

```diff
@@ -133,3 +133,14 @@ CAPTCHA_SECRET=
 WEBPUSH_SUBJECT=mailto:example@domain.com
 WEBPUSH_PUBLIC_KEY=
 WEBPUSH_PRIVATE_KEY=
+
+# Image processing
```

Comment:
Could you include new configs to docs? https://docs.intake24.org/config/api/

Maybe drop `APP_` prefix for consistently, as it's a different part of the config?